source/georef/georef.cpp

```diff
@@ -689,19 +692,19 @@ edge_t GeoRef::nearest_edge(const type::GeographicalCoord& coordinates,
     boost::optional<edge_t> res;
     float min_dist = 0., cur_dist = 0.;
     double coslat = ::cos(coordinates.lat() * type::GeographicalCoord::N_DEG_TO_RAD);
-    for (const auto& pair_coord : prox.find_within(coordinates, horizon)) {
+    for (const auto& ind : prox.find_within<proximitylist::IndexOnly>(coordinates, horizon, 30)) {
```

Comment:
Magic number spotted :eyes: 
Could you move the value `30` in a const variable named appropriately please :)

Reply:
doc added!!